src/piggui.rs

```diff
@@ -109,6 +107,7 @@ fn main() -> Result<(), iced::Error> {
 
     Piggui::run(Settings {
         window,
+        default_text_size: Pixels(14.0),
```

Comment:
OK, so you did do it here...
See my comments on discord... there still seems to be (on Pi) the same problem on the unsaved changes dialog..
Also, you can remove my code where I force the size on the pick_list for linux, as this *should* force that size too, no?